Description: fix for tap 15
Author: Yadd <yadd@debian.org>
Bug-Debian: https://bugs.debian.org/1009506
Forwarded: no
Last-Update: 2022-04-15

--- a/test/injection-leaking-1.js
+++ b/test/injection-leaking-1.js
@@ -28,7 +28,7 @@
 
 test('setup', function (t) {
   setup()
-  t.done()
+  t.end()
 })
 
 test('injection leaking at a distance', function (t) {
@@ -49,7 +49,7 @@
 })
 test('cleanup', function (t) {
   cleanup()
-  t.done()
+  t.end()
 })
 
 function setup () {
--- a/test/injection-leaking-2.js
+++ b/test/injection-leaking-2.js
@@ -29,7 +29,7 @@
 
 test('setup', function (t) {
   setup()
-  t.done()
+  t.end()
 })
 test('injection leaking at a distance', function (t) {
   t.plan(2)
@@ -49,7 +49,7 @@
 })
 test('cleanup', function (t) {
   cleanup()
-  t.done()
+  t.end()
 })
 
 function setup () {
--- a/test/injection-missing-1.js
+++ b/test/injection-missing-1.js
@@ -47,7 +47,7 @@
 
 test('setup', function (t) {
   setup()
-  t.done()
+  t.end()
 })
 
 test('injection leaking at a distance', function (t) {
@@ -68,7 +68,7 @@
 })
 test('cleanup', function (t) {
   cleanup()
-  t.done()
+  t.end()
 })
 
 function setup () {
--- a/test/injection-missing-2.js
+++ b/test/injection-missing-2.js
@@ -40,7 +40,7 @@
 
 test('setup', function (t) {
   setup()
-  t.done()
+  t.end()
 })
 
 test('injection leaking at a distance', function (t) {
@@ -64,7 +64,7 @@
 
 test('cleanup', function (t) {
   cleanup()
-  t.done()
+  t.end()
 })
 
 function setup () {
